src/main/java/net/jesteur/me/world/chunkgen/map/ImageUtils.java

```diff
@@ -0,0 +1,63 @@
+package net.jesteur.me.world.chunkgen.map;
+
+import java.awt.image.BufferedImage;
+import java.awt.image.ConvolveOp;
+import java.awt.image.DataBufferByte;
+import java.awt.image.Kernel;
+import java.util.Arrays;
+
+public class ImageUtils {
+    public static int BRUSH_SIZE = 24;
+    public static float ratio = 1.0f / (BRUSH_SIZE * BRUSH_SIZE);
```

Comment:
Since it's a const : 

```suggestion
    public static float RATIO = 1.0f / (BRUSH_SIZE * BRUSH_SIZE);
```